src/test/java/POM/pages/SongsPage.java

```diff
@@ -0,0 +1,30 @@
+package POM.pages;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+
+public class SongsPage extends BasePage{
+    By shuffleBtnLocator = By.cssSelector(".btn-shuffle-all");
+    By firstSongLocator = By.cssSelector(".play");
```

Comment:
Please, make all the locators private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example

    ```
    private By shuffleBtnLocator = By.cssSelector(".btn-shuffle-all");
    private By firstSongLocator = By.cssSelector(".play");
```

Reply:
Thank you for the feedback @Berichicko 👍🏽